internal/utils/context.go

```diff
@@ -0,0 +1,77 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+
+	azcorearm "github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+
+	"github.com/Azure/ARO-HCP/frontend/pkg/util"
```

Comment:
maybe pull this one as well so you don't end up adminapi->internal->frontend, I imagine that was not the intent